binary/arith.js

```diff
@@ -49,16 +49,19 @@ goog.provide('jspb.arith.UInt64');
  * @param {number} lo The low 32 bits.
  * @param {number} hi The high 32 bits.
  * @constructor
+ * @export
```

Comment:
this is an example of something that is only being exported for tests

can you open an issue to look into removing  export from APIs like this to improve size and just note that this will require a test refactoring.